Library/Formula/generate-json-schema.rb

```diff
@@ -0,0 +1,112 @@
+class GenerateJsonSchema < Formula
+  desc "Generate a JSON Schema from Sample JSON"
+  homepage "https://github.com/Nijikokun/generate-schema"
+  url "https://github.com/Nijikokun/generate-schema/archive/v2.1.1.tar.gz"
+  sha256 "bf43a7e616419876293e7de738dece27b3deb719cf5e8cc99bb309a5fb179af0"
+
+  head "https://github.com/Nijikokun/generate-schema.git"
+
+  depends_on "node"
+
+  def install
+    ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/npm/bin"
+
+    system "npm", "install"
+    libexec.install Dir["*"]
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    require "open3"
+
+    input = <<-EOS.undent
+      {
+          "id": 2,
+          "name": "An ice sculpture",
+          "price": 12.50,
+          "tags": ["cold", "ice"],
+          "dimensions": {
+              "length": 7.0,
+              "width": 12.0,
+              "height": 9.5
+          },
+          "warehouseLocation": {
+              "latitude": -78.75,
+              "longitude": 20.4
+          }
+      }
+    EOS
+
+    output =
```

Comment:
Wrap this in an `<<-EOS.undent` as you did above and then use a two-space indent from underneath that point. The bot will ignore things inside the `EOS` shield.